Library/DiscUtils.Ntfs/Bitmap.cs

```diff
@@ -201,6 +201,16 @@ public byte GetByte(long index)
             return 0;
         }
 
+        public int GetBytes(byte[] buffer, long index, int count)
```

Comment:
Also. Name the offset in `buffer` just that: `offset`. So we're consistent with others. :)